artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -226,6 +227,7 @@ public void start() throws Exception {
       if (isStarted()) {
          return;
       }
+      cleanupTemporaryFilesIfAnyBeforeStart();
```

Comment:
Can you just rename this to cleanupTmp(); ?

Reply:
I don't understand one thing.. can't you reuse the WebTmpCleaner here?

Reply:
It eventually calls WebTmpCleaner but before that we need to figure out where the tmp dir is. (because before the web server start there is no tmpdir in each web context. The tmpdir only
exists in the web context after the webapp (war) is deployed).